godot-ffi/src/toolbox.rs

```diff
@@ -183,7 +172,15 @@ pub(crate) type GetClassMethod = unsafe extern "C" fn(
     p_hash: sys::GDExtensionInt,
 ) -> sys::GDExtensionMethodBindPtr;
 
-pub type ClassMethodBind = sys::GDExtensionMethodBindPtr;
+#[derive(Clone, Copy)]
+pub struct ClassMethodBind(pub sys::GDExtensionMethodBindPtr);
+
+// SAFETY: `sys::GDExtensionMethodBindPtr` is effectively the same as a `unsafe extern "C" fn`. So sharing it between
+// threads is fine, as using it in any way requires `unsafe` and it is up to the caller to ensure it is thread safe
+// to do so.
+unsafe impl Sync for ClassMethodBind {}
+// SAFETY: See `Sync` impl safety doc.
+unsafe impl Send for ClassMethodBind {}
```

Comment:
I assume the point of this abstraction is to add `Send`/`Sync`, which the raw pointer doesn't have?

If yes, maybe add this as a comment above the struct.